frontend/components/panels/WalletDepositCheckPanel.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>{{ currentStateAndTitle.title }}</template>
+        <div class="flex flex-col gap-2">
+            <TextBlock v-if="currency === 'MKR'">
```

Comment:
Why does it only shown for the MKR? If it's about the table with details, you should better introduce `isTableShown` prop for example